### PR TITLE
Create a stub of the file for laverne.md

### PR DESCRIPTION
**Pull request by bensumali**
#59

### CODE REVIEW

File: laverne.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (githubteacher) [inline laverne.md:2]: It would be awesome to know places to eat in your hometown!

**Comment by loupendley** [comment]
This looks excellent!

File: laverne.md (hunk 1 above)
Comment (iamdv) [inline laverne.md:2]: Vapiano is best italian place

**Comment by baharehtraining** [comment]
This is a comment

**Pull request merged**